packages/lib-classifier/src/components/Classifier/Classifier.js

```diff
@@ -49,6 +49,8 @@ export default function Classifier({
   useEffect(function onWorkflowVersionChange() {
     const { workflows, subjects } = classifierStore
     if (workflowSnapshot) {
+      const { subjectSet } = workflows.active
+      workflowSnapshot.subjectSet = subjectSet
```

Comment:
```suggestion
      const subjectSet = workflows?.active?.subjectSet
      if (subjectSet) {
        workflowSnapshot.subjectSet = subjectSet
      }
```

Getting `Uncaught TypeError: Cannot read properties of undefined (reading 'subjectSet')` when testing on https://local.zooniverse.org:8080/?project=nora-dot-eisner/planet-hunters-tess&env=production&demo=true. 

The suggestion, or whatever similar change you see fit, will fix TESS while keeping the fix for HMS NHS.

Reply:
I've rewritten this so that the data flow for hydration is the same pattern used in NextJS:
component props => workflow data snapshot => classifier workflow store.